site/specs-temp/numbers.json

```diff
@@ -19037,57 +18848,360 @@
+            }
+          }
+        },
+        "responses": {
+          "200": {
+            "description": "<p>a 200 OK is presumed to be acceptance of the API call.  In the case of the notification callback API the response code and any returned message is stored in the &lt;Status&gt; element of the &lt;Subscription&gt;</p>\n"
+          },
+          "400": {
+            "description": "<p>Anything other than a 200 OK is presumed to be a failure of the API call.  In the case of the notification callback API the response code and any returned message is stored in the &lt;Status&gt; element of the &lt;Subscription&gt;.  No subsequent action is taken is based on the error.</p>\n"
+          }
+        }
+      }
+    },
+    "/sippeers/bulk/accounts/{accountId}/sites/{siteId}/sippeers/{sipPeerId}/movetns": {
+      "post": {
+        "tags": [
+          "Tns"
+        ],
+        "description": "The POST method moves all telephone numbers specified in the body to the given SIP peer.<br>The source SIP peer is determined by the Telephone Number, i.e. the PUT method can move multiple numbers from different source SIP peers.<br>The destination SIP peer is specified in the URL.<br>NOTE: only a maximum of 5000 Telephone Numbers can be moved in one operation.",
+        "operationId": "CreateSipPeerTelephoneNumberBulk",
+        "summary": "Move bulk tns to sipper",
+        "parameters": [
+          {
+            "$ref": "#/components/parameters/AccountIdPathParam"
+          },
+          {
+            "$ref": "#/components/parameters/SiteIdPathParam"
+          },
+          {
+            "$ref": "#/components/parameters/SipPeerIdPathParam"
+          }
+        ],
+        "requestBody": {
+          "content": {
+            "application/xml": {
+              "schema": {
+                "$ref": "#/components/schemas/SipPeerTelephoneNumberBulk"
+              }
+            }
+          }
+        },
+        "responses": {
+          "200": {
+            "description": "The request has been successfully received.",
+            "content": {
+              "application/xml": {
+                "schema": {
+                  "$ref": "#/components/schemas/SipPeerTelephoneNumberBulkResponse"
+                }
+              }
+            }
+          },
+          "400": {
+            "description": "Bad Request. Possible errors in response:<ul>    <li>Shared TNs with messaging cannot be moved to a sippeer without the SMS feature enabled.</li>    <li>Shared TNs with voice cannot be moved to a sippeer with the SMS feature enabled.</li></ul>",
+            "content": {
+              "application/xml": {
+                "schema": {
+                  "$ref": "#/components/schemas/SipPeerTelephoneNumberBulkErrorResponse"
+                }
+              }
+            }
+          }
+        }
+      }
+    },
+    "/sippeers/products/termination/accounts/{accountId}/sites/{siteId}/sippeers/{sipPeerId}/products/termination": {
```

Comment:
is this the right path? Shouldn't it be `/accounts/{accountId}/sites/{siteId}/sippeers/{sipPeerId}/products/termination`

Reply:
Yeah, thanks @mrmeyers99, I missed the url-rewrite logic here, addressing